public/css/history.css

```diff
@@ -64,21 +79,17 @@
 	padding: 25px 50px 50px;
 }
 
-.section-header {
-	width: 100%;
-	height: 35px;
-	border-bottom: 1px solid black;
-	text-align: center;
-}
-
 #twenty,
 #eighteen {
-	background-color: #d1ebfa;
+	background-color: #191a1b;
+	color: #ffffff;
```

Comment:
Please swap out your hex codes in all of your CSS for the CSS variables provided from `style.css`. It's more consistent and easier for you in the long run.